README.md

```diff
@@ -9,17 +9,30 @@ This is the Glitch equivalent of running `create-probot-app` to generate a new p
 
 To get your own Glitch-hosted Probot up-and-running:
 
-1. Head on back to our [Devlopment Docs](https://probot.github.io/docs/development/#configuring-a-github-app) to configure your GitHub App.
 
-2. Click the **New File** button (at left on Glitch) and type `.data/private-key.pem`. Then click **Add File**. Open the private key you downloaded during your Github App creation, and copy/paste the contents into your new file.
+1. [Configure a new app on Github](https://github.com/settings/apps/new).
+    - Hit the "Show" button on the top left of this page to find the URL. It will look something like `https://random-word.glitch.me/probot`, except the domain will be specific to your app.
+    - For the Homepage URL, use your app URL that you just found (updating the domain to match yours): `https://random-word.glitch.me/probot`
```

Comment:
Homepage URL is not a required field, so I don't think this is necessary.

Reply:
👍